wagtail/admin/templates/wagtailadmin/shared/headers/slim_header.html

```diff
@@ -79,14 +79,35 @@ <h1 class="w-sr-only">
                         {% endif %}
 
                         {% if filters %}
-                            {% fragment as filters_icon %}{% icon name="sliders" title=_("Show filters") %}{% endfragment %}
-                            {% dialog_toggle classname="w-filter-button" dialog_id="filters-dialog" text=filters_icon %}
-
-                            {% dialog theme="floating" id="filters-dialog" title=_("Filters") dialog_root_selector="[data-search-form]" %}
-                                {% for field in filters.form %}
-                                    {% formattedfield field %}
-                                {% endfor %}
-                            {% enddialog %}
+                            <div class="w-drilldown" data-controller="w-drilldown" data-action="w-swap:success@document->w-drilldown#updateParamsCount">
+                                {% fragment as toggle_suffix %}
+                                    <span class="w-drilldown__count" hidden></span>
+                                {% endfragment %}
+                                {% dropdown theme="drilldown" toggle_icon="sliders" toggle_classname="w-filter-button" toggle_aria_label=_("Show filters") toggle_suffix=toggle_suffix %}
+                                    <div class="w-drilldown__contents">
+                                        <div class="w-drilldown__menu" data-w-drilldown-target="menu">
+                                            <h2 class="w-help-text w-pl-5 w-py-2.5 w-my-0">{% trans "Filter by" %}</h2>
+                                            {% for field in filters.form %}
+                                                <button class="w-drilldown__toggle" data-w-drilldown-target="toggle" data-action="click->w-drilldown#open" type="button" class="w-flex w-justify-between" aria-expanded="false" aria-controls="drilldown-{{field.auto_id}}">
```

Comment:
Nitpicky but we do talk about moving data attributes to the last attributes in our documentation.

This makes it easier to skim through the HTML to find data attributes and non.

```suggestion
                                                <button class="w-drilldown__toggle" type="button" class="w-flex w-justify-between" aria-expanded="false" aria-controls="drilldown-{{field.auto_id}}" data-w-drilldown-target="toggle" data-action="click->w-drilldown#open" >
```